### DISCUSSION TITLE
git move doesn't move the branch pointer after rebase

### DISCUSSION THREAD

**Discussion by yagnaa** [arxanas/git-branchless#401] [General]
`git move -d branch_B  -b branch_A`
there is a conflict in the last commit of branch_B. 
So run 
`git move -d branch_B  -b branch_A --merged`
in memory rebase fails
on disk rebase is attempted
$vi <conflictfile> (fix)
$ git add *
$ git rebase --continue

Expectation;
git move detects the rebase completion and moves the branch_B appropriately.

Behaviour:
After rebase we are left in detached HEAD state and have to move the branch_B to the detached HEAD manually..

**Comment by arxanas** [comment]
Hi @yagnaa, thanks for reporting. The next time you see this, can you file a bug using the issue tracker? In particular, the output of `git branchless bug-report` might be useful.
